docker/focal/Dockerfile

```diff
@@ -29,19 +29,22 @@ ENV MINIFI_HOME $MINIFI_BASE_DIR/nifi-minifi-cpp-$MINIFI_VERSION
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
 
 RUN apt update
-RUN DEBIAN_FRONTEND="noninteractive" apt install -y openjdk-8-jdk openjdk-8-source python3.9-dev sudo git maven autogen autoconf automake cmake
+RUN DEBIAN_FRONTEND="noninteractive" apt install -y openjdk-8-jdk openjdk-8-source python3.9-dev sudo git maven autogen autoconf automake cmake software-properties-common
 
 RUN mkdir -p $MINIFI_BASE_DIR
```

Comment:
These commands could be merged into a single RUN command

Reply:
Would it be better that way?

Reply:
Usually it's recommended to minimize the layer count and as every command in a Dockerfile creates a new layer it is good practice to merge multiple RUN commands to have less layers.

Reply:
Does this include ENV and COPY, or only RUN?

Reply:
Only RUN commands can be chained in a bash like manner with `&&` ENV and COPY don't really have a similar option

Reply:
I mean do they also introduce separate layers? Should I keep DEBIAN_FRONTEND as a prefix to the command or separate it to an ENV command?

Reply:
But if the question was regarding the creation of the layers, yes all of those create new layers.

Reply:
I think keeping it as a prefix is better.